flake.nix

```diff
@@ -30,9 +35,11 @@
         ./validation/demo-plutarch/build.nix
         ./validation/demo-plutustx/build.nix
         ./transactions/demo-ctl/build.nix
+        ./transactions/demo-rust/build.nix
       ];
       debug = true;
-      systems = [ "x86_64-linux" "x86_64-darwin" ];
+      systems = [ "x86_64-linux" ]; # TODO(szg251): fix darwin CI build and add "x86_64-darwin"
```

Comment:
Funnily (yeah, it wasn't that funny) I could build and run tests on a local x86_64-darwin machine, but it fails on CI.
It looks like there's an upstream issue with plutip and ogmios